<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleKinectsDialog</class>
 <widget class="QDialog" name="MultipleKinectsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="kinects_tw">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="enable_kinect_cb">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="send_osc_cb">
           <property name="text">
            <string>Send OSC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="osc_settings_b">
           <property name="text">
            <string>OSC settings...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Detection method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="detection_method_cb_b"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Base image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="base_image_cb_b"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Overlay</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="centroids_cb">
           <property name="text">
            <string>Centroids</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="boundaries_cb">
           <property name="text">
            <string>Boundaries</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="base_image_lab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>kinects_tw</tabstop>
  <tabstop>enable_kinect_cb</tabstop>
  <tabstop>send_osc_cb</tabstop>
  <tabstop>osc_settings_b</tabstop>
  <tabstop>detection_method_cb_b</tabstop>
  <tabstop>base_image_cb_b</tabstop>
  <tabstop>centroids_cb</tabstop>
  <tabstop>boundaries_cb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
